README.md

```diff
@@ -1,7 +1,14 @@
-# remix
+# REMIX
 Ethereum IDE and tools for the web
 
-INSTALLATION:
+## REMIX WEBSITE:
+
+Remix is avalaible at https://ethereum.github.io/remix
+You'll have to run your own node using the following parameters:
+geth --rpc --rpcapi 'web3,eth,debug' --rpcport 8545 --rpccorsdomain '*'
+geth will run the rpc server on http://localhost:8545, remix uses this url by default to connect to web3.
+
```

Comment:
Perhaps add some warnings - only use unlock any keys, do not use that geth instance together with the wallet, ...